### PR TITLE
Implement Styled Components 

### PR DESCRIPTION
**Pull request by willresen**
In this PR:

- Use styled components instead of a stylesheet
- Fix unit tests that broke from this transition and from remove babel-transform-runtime in 
4270544
- Refactor navbar logic to use native React methods instead of JQuery
- Separate seeder.jsx into several files
- Switch ESLint style guide to AirBnB

TODO:
- Refactor the database seeder
- Migrate media assets to AWS
- Integration and E2E tests

Possible upgrades:
- Continuous integration w/ Circle CI

### CODE REVIEW

File: client/src/components/Movie.jsx [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
+        <RoundedText>{movie.runtime}</RoundedText>
+      </Container>
+      <RatingContainer margin={true}>
+        <RatingContainer margin={false} hover={true}>
+          <img src="https://fec1-arwen.s3.amazonaws.com/infoplate/Star_empty.svg"></img>
+          <FullStars width={Number(movie.vudu_rating)}>
+            <img src="https://fec1-arwen.s3.amazonaws.com/infoplate/Star_full.svg"></img>
+          </FullStars>
+        </RatingContainer>
+        <img src={`${movie.rt_rating > 50 ? 'https://fec1-arwen.s3.amazonaws.com/infoplate/Tomato_certified_fresh.svg' : 'https://fec1-arwen.s3.amazonaws.com/infoplate/Tomato_rotten.svg'}`} style={{ display: 'flex', transform: 'scale(.8)' }}></img>
```
Comment (HBMattoon) [inline client/src/components/Movie.jsx:20]: are you planning on also showing if it's just normal fresh or if it's rotten?

File: client/src/components/MoviesCarousel.jsx [hunk 2]
```diff
@@ -31,28 +32,29 @@ class MovieCarousel extends React.Component {
...
-          <div id="movies_carousel" style={{ transform: `translate3d(${this.state.position}px, 0px, 0px)` }}>
-            {this.state.movies.map(movie => <div className="movie"><Movie movie={movie} /></div>)}
-          </div>
-          <div id="m_forward" onClick={this.moveForward}></div>
-        </div>
-      </div>
+      <Container>
+        <Title>Movies Featuring {this.props.selectedPersonnel}</Title>
+        <CarouselContainer>
+          <NavBackward onClick={this.moveBackward} style={{height: '202px'}} />
```
Comment (HBMattoon) [inline client/src/components/MoviesCarousel.jsx:49]: can you add the height:202px to your styling.jsx file?

File: client/src/components/MoviesCarousel.jsx [hunk 3]
```diff
@@ -31,28 +32,29 @@ class MovieCarousel extends React.Component {
...
+          <Carousel ref="MoviesCarousel" position={this.state.position}>
+            {this.state.movies.map(movie => <Plate><Movie movie={movie} /></Plate>)}
+          </Carousel>
+          <NavForward onClick={this.moveForward} style={{top: '354px', height: '202px'}}/>
```
Comment (HBMattoon) [inline client/src/components/MoviesCarousel.jsx:53]: same as with navBackward. can you move your style to styling.jsx

**Review by HBMattoon — APPROVED** [review]
lgtm, the styled components really make your components more readable. Nice job!

File: client/src/components/Movie.jsx (hunk 1 above)
Comment (willresen) [reply client/src/components/Movie.jsx:20]: @HBMattoon I am using a ternary operator on line 20 to distinguish the two!

File: client/src/components/MoviesCarousel.jsx (hunk 3 above)
Comment (willresen) [reply client/src/components/MoviesCarousel.jsx:53]: @HBMattoon Good catch - will move these over to styling.jsx in the next PR.

**Pull request merged**